.pre-commit-config.yaml

```diff
@@ -0,0 +1,23 @@
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+repos:
+- repo: https://github.com/astral-sh/ruff-pre-commit
+  rev: v0.1.14
+  hooks:
+    # Run the linter.
+  - id: ruff
+    # Run the formatter.
+  - id: ruff-format
+    args: [--check]
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v4.5.0
+  hooks:
```

Comment:
Is it possible to add a line-ending fixer? (only `LF`, no `CR` or `CRLF`)

Reply:
Yes: https://github.com/pre-commit/pre-commit-hooks?tab=readme-ov-file#mixed-line-ending
I'll add it with --fix=no